### DISCUSSION TITLE
WPF 教程网站推荐

### DISCUSSION THREAD

**Discussion by BYJRK** [BYJRK/DotNet-Discussions#7] [WPF 开发]
- [2,000 Things You Should Know About WPF](https://wpf.2000things.com/)
- [The complete WPF tutorial](https://wpf-tutorial.com/)
- [适用于 WPF 和 .NET 上的 Windows 窗体的桌面指南](https://learn.microsoft.com/zh-cn/dotnet/desktop/?view=netdesktop-6.0)
- 其他待补充……

**Comment by LiuChao19940201** [comment]
https://search.bilibili.com/all?keyword=%E5%8D%81%E6%9C%88%E7%9A%84%E5%AF%92%E6%B5%81&from_source=webtop_search&spm_id_from=333.1007&search_source=5
